examples/http-server-using-redis/main_test.go

```diff
@@ -55,6 +63,12 @@ func TestHTTPServerUsingRedis(t *testing.T) {
 }
 
 func TestRedisSetHandler(t *testing.T) {
+	metricsPort := testutil.GetFreePort(t)
+	httpPort := testutil.GetFreePort(t)
+
+	t.Setenv("METRICS_PORT", fmt.Sprint(metricsPort))
+	t.Setenv("HTTP_PORT", fmt.Sprint(httpPort))
+
```

Comment:
Why alternating between strconv.Itoa and fmt.Sprint? (The previous files vs this one and next ones)